examples/drivers/windows/otLwf/filter.h

```diff
@@ -36,8 +36,11 @@
 #define _FILT_H
 
 // The maximum allowed addresses an OpenThread interface
-#define OT_MAX_ADDRESSES 10
-#define OT_MAX_AUTO_ADDRESSES (OT_MAX_ADDRESSES - 4)
+#if OPENTHREAD_ENABLE_DHCP6_CLIENT
```

Comment:
So you are saying that if `DHCP6_SERVER` is enabled, then I need to account for `OPENTHREAD_CONFIG_NUM_DHCP_PREFIXES` ALOCs?